test/e2e/manifest_test.go

```diff
@@ -405,4 +406,41 @@ var _ = Describe("Podman manifest", func() {
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(Exit(0))
 	})
+
+	It("manifest rm should not remove image if it resolves to image instead of a manifest", func() {
+		// build a container image
+		imgName := "testimg"
+		dockerfile := fmt.Sprintf(`FROM %s
+RUN echo hello`, ALPINE)
+		podmanTest.BuildImage(dockerfile, imgName, "false")
```

Comment:
I think we can just do `manifest rm ALPINE` instead, can we?  This would speed up the test as we can remove the build.